src/tests/baseservices/RuntimeConfiguration/TestConfig.cs

```diff
@@ -64,6 +64,9 @@ static int Verify_ServerGC_Env_Override_Prop(string[] _)
 
     static int Main(string[] args)
     {
+        ClearEnvironmentVariable("DOTNET_gcServer");
```

Comment:
I think we should only do this in the `RunTests()` method or perhaps before we call it within the branch below.

Reply:
Moved to `RunTests()`.